src/main/java/Library.java

```diff
@@ -312,28 +312,42 @@ public static int httpGet(URL address) throws IOException {
         HttpURLConnection con = (HttpURLConnection) address.openConnection();
         return con.getResponseCode();
     }
-    
-    /**
-     * Checks if given number is a prime number
-     * Prime number is a number that is greater than 1 and divided by 1 or itself only
-     * Credits: https://en.wikipedia.org/wiki/Prime_number
-     * @param number number to check prime
-     * @return true if prime
+	
+	/**
+     * Zip an entire directory
+     * @param srcDirName the filename of the source directory
+     * @param zipDirName the filename of the destination zip file
+	 * @throws IOException
      */
-	public static boolean isPrime(int number) {
```

Comment:
I believe this is an unintentional change but please don't remove the existing methods.